<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureFloorToolWidget</class>
 <widget class="QWidget" name="MeasureFloorToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_99_0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_99_1">
       <item>
        <widget class="ImageGrid" name="m_imageGrid" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="measurementsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2_0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_1">
           <property name="margin">
            <number>0</number>
           </property>

           <item>
	        <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Dimensions</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QFormLayout" name="formLayout_2">

		        <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Plan Size:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="m_planSizeEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>

                <item row="1" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Mask Size:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="m_maskSizeEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>

			   </layout>
              </item>
             </layout>
            </widget>
           </item>

           <item>
	        <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Locations</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QFormLayout" name="formLayout_2">

		        <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Start Point:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="m_startPointEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>

                <item row="1" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>End Point:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="m_endPointEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>

                <item row="3" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Distance:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="m_distanceEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>

			   </layout>
              </item>
             </layout>
            </widget>
           </item>

           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>

           <item>
            <widget class="QCheckBox" name="m_overlayMask">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Overlay Mask</string>
             </property>
            </widget>
           </item>

          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomImageView</class>
   <extends>QWidget</extends>
   <header>ZoomImageView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
